internal/os/volume/api.go

```diff
@@ -109,6 +110,27 @@ func (VolAPIImplementor) ResizeVolume(volumeID string, size int64) error {
 		finalSize = size
 	}
 
+	cmd = fmt.Sprintf("Get-Volume -UniqueId \"%s\" | Get-partition | Select Size", volumeID)
```

Comment:
could you add a separate func like `func getVolumeSize(id) (int64, error)`, that will make the logic more clear.